src/current/_includes/v24.2/cdc/sink-list.md

```diff
@@ -1,3 +1,4 @@
+- {% if page.name == "changefeed-sinks.md" %} [Amazon MSK Serverless](#amazon-msk-serverless) {% else %} [Amazon MSK Serverless]({% link {{ page.version.version }}/changefeed-sinks.md %}#amazon-msk-serverless) {% endif %}
```

Comment:
question: Right now, I've place MSK Serverless as its own supported sink. That OK? The alternative is to have it as a sub-bullet under Kafka?

I also have an issue to add plain MSK to the sink list, so that is a consideration too.

Reply:
makes more sense to document msk/msk serverless as the same sink under 'Amazon MSK'